akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -380,6 +380,10 @@ sealed trait SslTlsOutbound
  *  - `enabledProtocols` will be passed to `SSLEngine::setEnabledProtocols()`
  *  - `clientAuth` will be passed to `SSLEngine::setWantClientAuth()` or `SSLEngine.setNeedClientAuth()`, respectively
  *  - `sslParameters` will be passed to `SSLEngine::setSSLParameters()`
+ *
+ * Please note that passing `clientAuth = None` means that no change is done
```

Comment:
Wouldn't it be less error-prone if it wasn't an option, and the None behaviour was encoded as a ClientAuth?